testNetworkPartitionThenRecovery test fails on travis

[ERROR] Failures: 
[ERROR]   MembershipProtocolTest.testNetworkPartitionThenRecovery:99->assertTrusted:612 Expected 1 trusted members [0.0.0.0:46437], but actual: [0.0.0.0:46535, 0.0.0.0:46437] ==> expected: <1> but was: <2>

--- a/cluster/src/main/java/io/scalecube/cluster/ClusterMath.java
+++ b/cluster/src/main/java/io/scalecube/cluster/ClusterMath.java
@@ -130,7 +130,7 @@ public final class ClusterMath {
    * @param num num
    * @return ceil(log2(n + 1))
    */
-  public static int ceilLog2(int num) {
+  private static int ceilLog2(int num) {
     return 32 - Integer.numberOfLeadingZeros(num);
   }
 }

--- a/cluster/src/test/java/io/scalecube/cluster/membership/MembershipProtocolTest.java
+++ b/cluster/src/test/java/io/scalecube/cluster/membership/MembershipProtocolTest.java
@@ -34,10 +34,12 @@ import reactor.core.scheduler.Schedulers;
 public class MembershipProtocolTest extends BaseTest {
 
   public static final Duration TIMEOUT = Duration.ofSeconds(10);
+
   public static final int TEST_SYNC_INTERVAL = 500;
   public static final int TEST_SYNC_TIMEOUT = 100;
   public static final int TEST_SUSPICION_MULT = 3;
   private static final int TEST_PING_INTERVAL = 200;
+
   private Scheduler scheduler;
 
   @BeforeEach

--- a/cluster/src/test/java/io/scalecube/cluster/membership/MembershipProtocolTest.java
+++ b/cluster/src/test/java/io/scalecube/cluster/membership/MembershipProtocolTest.java
@@ -98,8 +98,9 @@ public class MembershipProtocolTest extends BaseTest {
 
     try {
 
+      int suspicionMult = ClusterConfig.DEFAULT_SUSPICION_MULT;
       long suspicionTimeoutSec =
-          ClusterMath.suspicionTimeout(ClusterConfig.DEFAULT_SUSPICION_MULT, 3, TEST_PING_INTERVAL) / 1000;
+          ClusterMath.suspicionTimeout(suspicionMult, 3, TEST_PING_INTERVAL) / 1000;
       awaitSeconds(suspicionTimeoutSec + 2);
 
       assertTrusted(cmA, a.address());

--- a/cluster/src/test/java/io/scalecube/cluster/membership/MembershipProtocolTest.java
+++ b/cluster/src/test/java/io/scalecube/cluster/membership/MembershipProtocolTest.java
@@ -33,10 +33,11 @@ import reactor.core.scheduler.Schedulers;
 
 public class MembershipProtocolTest extends BaseTest {
 
-  private static final int TEST_PING_INTERVAL = 200;
-
   public static final Duration TIMEOUT = Duration.ofSeconds(10);
-
+  public static final int TEST_SYNC_INTERVAL = 500;
+  public static final int TEST_SYNC_TIMEOUT = 100;
+  public static final int TEST_SUSPICION_MULT = 3;
+  private static final int TEST_PING_INTERVAL = 200;
   private Scheduler scheduler;
 
   @BeforeEach
@@ -95,7 +96,10 @@ public class MembershipProtocolTest extends BaseTest {
     c.networkEmulator().block(members);
 
     try {
-      awaitSeconds(3);
+
+      long suspicionTimeoutSec =
+          ClusterMath.suspicionTimeout(TEST_SUSPICION_MULT, 3, TEST_PING_INTERVAL) / 1000;
+      awaitSeconds(suspicionTimeoutSec + 2);
 
       assertTrusted(cmA, a.address());
       assertNoSuspected(cmA);
@@ -108,7 +112,7 @@ public class MembershipProtocolTest extends BaseTest {
       b.networkEmulator().unblockAll();
       c.networkEmulator().unblockAll();
 
-      awaitSeconds(3);
+      awaitSeconds(TEST_SYNC_INTERVAL * 2 / 1000);
 
       assertTrusted(cmA, a.address(), b.address(), c.address());
       assertNoSuspected(cmA);
@@ -539,8 +543,9 @@ public class MembershipProtocolTest extends BaseTest {
     // Create faster config for local testing
     return ClusterConfig.builder()
         .seedMembers(seedAddresses)
-        .syncInterval(2000)
-        .syncTimeout(1000)
+        .syncInterval(TEST_SYNC_INTERVAL)
+        .syncTimeout(TEST_SYNC_TIMEOUT)
+        .suspicionMult(TEST_SUSPICION_MULT)
         .pingInterval(TEST_PING_INTERVAL)
         .pingTimeout(100)
         .metadataTimeout(100);

